src/components/PageWrapper.js

```diff
@@ -32,7 +33,8 @@ const PageWrapper = ({ children }) => {
         }, 'Saturn')
       ]),
       a({ href: Nav.getLink('privacy'), style: styles.link }, 'Privacy Policy'),
-      a({ target: '_blank', href: tosUrl, style: styles.link }, 'Terms of Service')
+      a({ target: '_blank', href: tosUrl, style: styles.link }, 'Terms of Service'),
+      div({ style: { marginLeft: 'auto' } }, new Date(SATURN_BUILD_TIMESTAMP).toLocaleString())
```

Comment:
This looks like it is telling me the current time of day, except that it's wrong. Could we see about some sort of indicator that this is the build time rather than the time of day?